web/language/models.py

```diff
@@ -79,12 +79,47 @@ class LanguageMember(models.Model):
     user = models.ForeignKey(User, on_delete=models.CASCADE)
 
 
+class PlaceNameCategory(BaseModel):
+    icon_name = models.CharField(max_length=32, null=True, default=None)
+
+
 class PlaceName(BaseModel):
     point = models.PointField(null=True, default=None)
     other_names = models.CharField(max_length=255, default="")
     audio_file = models.FileField(null=True, blank=True)
     kind = models.CharField(max_length=15, default="")
 
+    category = models.ForeignKey(PlaceNameCategory, on_delete=models.SET_NULL, null=True)
+    western_name = models.CharField(max_length=64, blank=True)
```

Comment:
@DenisCalixto because we have the BaseModel as a parent, there's also a "name" field. Perhaps we'd use that as the western name, or stop inheriting BaseModel? Either, or.